pkg/splunk/enterprise/clustermaster.go

```diff
@@ -144,6 +152,15 @@ func ApplyClusterMaster(client splcommon.ControllerClient, cr *enterpriseApi.Clu
 		if err != nil {
 			scopedLog.Error(err, "Error in deleting automated monitoring console resource")
 		}
+		//Update MC configmap
+		if cr.Spec.MonitoringConsoleRef.Name != "" {
```

Comment:
Possibly a dumb question but do we need to do anything is an IDC CR removes a MC ref?  Are we handling that case?

Reply:
if the mcref is changed/removed we have a story to cover that scenario CSPL-1087